tests/test_utils/utils.py

```diff
@@ -1,42 +1,55 @@
 import json
 import os
 
+CONF_FILE = os.environ.get("EXODUS_LAMBDA_CONF_FILE")
 
-def generate_test_config(conf="configuration/lambda_config.json"):
-    with open(conf, "r") as json_file:
-        conf = json.load(json_file)
-
-    # table
-    conf["table"]["name"] = "test-table"
-    conf["table"]["available_regions"] = [
-        "us-east-1",
-        "us-east-2",
-        "us-west-1",
-        "us-west-2",
-        "ca-central-1",
-        "eu-central-1",
-        "eu-west-2",
-        "eu-west-3",
-        "eu-west-1",
-        "sa-east-1",
-        "ap-south-1",
-        "ap-northeast-2",
-        "ap-southeast-1",
-        "ap-southeast-2",
-        "ap-northeast-1",
-    ]
-
-    # logging
-    conf["logging"]["formatters"]["default"][
-        "format"
-    ] = "[%(levelname)s] - %(message)s\n"
-    conf["logging"]["loggers"]["origin-response"]["level"] = "DEBUG"
-    conf["logging"]["loggers"]["origin-request"]["level"] = "DEBUG"
-    conf["logging"]["loggers"]["default"]["level"] = "DEBUG"
-
-    conf["config_table"]["name"] = "test-config-table"
-    conf["config_table"]["cache_ttl"] = 2
 
+def generate_test_config(conf="configuration/lambda_config.json"):
```

Comment:
This doesn't seem very useful now that we're setting up a fake config file in conftest. Shouldn't we just be using that for all tests?

The only difference seems to be the table's available_regions. We can just add the 5 region names to the conftest config.

Reply:
+1, yeah, keep the conf consistent.